deepcell/image_generators/semantic.py

```diff
@@ -117,14 +122,15 @@ def __init__(self,
         self.save_format = save_format
         self.min_objects = min_objects
 
+        self.x = np.asarray(X)
+        self.y = np.asarray(y)
+
```

Comment:
What is this for? If they're not already arrays, wouldn't it crash on the dtype check above

Reply:
I have no idea. Initially I just deleted it (it was on line 109 before), but I got some pretty low-level error messages about index not being a numpy array when I did. So somewhere, something is generating a list.